content/en/docs/concepts/traffic-management/index.md

```diff
@@ -13,237 +13,143 @@ aliases:
+none depending on your use case.
+
+### Why use virtual services? {#why-use-virtual-services}
+
+Virtual services play a key role in making Istio’s traffic management flexible
+and powerful. They do this by strongly decoupling where clients send their
+requests from the destination workloads that actually implement them. Virtual
+services also provide a rich way of specifying different traffic routing rules
+for sending traffic to those workloads.
+
+So why is this so useful? Without virtual services, Envoy just distributes
+traffic using round-robin load balancing between all service instances, as
+described in the introduction. You can improve this behavior with what you know
+about the workloads: for example, some might represent a different version. This
+can be useful in A/B testing where you might want to configure traffic routes
+based on percentages across different service versions, or if you want to direct
+traffic from your internal users to a particular set of instances.
+
+With a virtual service, you can specify traffic behavior for one or more hostnames.
+You use routing rules in the virtual service that tell Envoy how to send the
+virtual service’s traffic to appropriate destinations. Route destinations can
+be versions of the same service or entirely different services.
+
+A typical use case is to send traffic to different versions of a service,
+specified as service subsets. Clients send requests to the virtual service host as if
+it was a single entity, and Envoy then routes the traffic to the different
+versions depending on the virtual service rules (for example "20% of calls go to
+the new version" or "calls from these users go to version 2"). This allows you to,
+for instance, create a canary rollout where you gradually increase the
+percentage of traffic that’s sent to a new service version. The traffic routing
+is completely separate from the instance deployment, meaning that the number of
+instances implementing the new service version can scale up and down based on
+traffic load without referring to traffic routing at all. By contrast, container
+orchestration platforms like Kubernetes only support traffic distribution based
+on instance scaling, which quickly becomes complex. You can read more about how
+virtual services help with canarying in [Canary Deployments using Istio](/blog/2017/0.1-canary/).
+
+Virtual services also let you:
+
+-   Address multiple application services through a single virtual service. If
+    your mesh uses Kubernetes, for example, you can configure a virtual service
+    to handle all services in a specific namespace. Mapping a single
+    virtual service to multiple "real" services can be particularly useful in
+    facilitating turning a monolithic application into a composite service built
+    out of distinct microservices without requiring the consumers of the service
+    to adapt to the transition: your rules can specify "calls to these URIs of
```

Comment:
change : to period.  Your rules which type of rules?

Reply:
done.